index.html

```diff
@@ -2,59 +2,223 @@
+            <li><a href="#contact">Contacts</a></li>
+          </ul>
+        </div>
+
+    </nav>
+
+    <section class="head-section">
+
+      <h1 class="header-title">I'm Collins <br class="desk-break"> Glad to see you!</h1>
+
+      <p class="header-text">
+        I'm a software developer! I can help you build a product ,
+        feature or website Look through some of my work and experience! If you
+        like what you see and have a project you need coded,
+        don't hestiate to contact me.
+      </p>
+
+      <p class="connect">Let's connect</p>
+
+      <div class="social-icon">
+        <ul>
+          <li><a href="https://twitter.com/"><img src="./images/twitter.png" alt="Twitter Icon"></a></li>
+          <li><a href="https://www.linkedin.com/in/"><img src="./images/linkedin.png" alt="Linkedin Icon"></a></li>
+          <li><a href="#"><img src="./images/medium.png" alt="Medium Icon"></a></li>
+          <li><a href="https://github.com/"><img src="./images/github.png" alt="GitHub Icon"></a></li>
+          <li><a href="#"><img src="./images/Fingers icon.png" alt="Fingers Icon"></a></li>
+        </ul>
+      </div>
+  </header>
+
+  <section class="work-section">
+      <div class="achievement">
+      <div class="img">
+        <img src="./images/Desktop3.png" alt="" srcset="">
+      </div>
+      <article>
+        <h3>Tonic</h3>
+        <p class="infos">
+          <span>canopy</span> • Back End Dev • 2015
+        </p>
+        <p class="prg">A daily selection of privately personalized reads; no accounts or sign-ups required..</p>
+        <ul>
+          <li>HTML</li>
+          <li>XML</li>
+          <li>CSS</li>
+        </ul>
+        <a href="#" class="btn-text">Get my resume</a>*
+      <br> 
+      &nbsp;
+      </article>
+    </div>
+
+    <div class="achievement">
+      <div class="img">
+        <img src="./images/Desktop-4.png" alt="" srcset="">
+      </div>
+      <article>
+        <h3>Multi-Post Stories</h3>
+        <p class="infos">
+          <span>canopy</span> • Back End Dev • 2015
+        </p>
+        <p class="prg">A daily selection of privately personalized reads; no accounts or sign-ups required..</p>
+        <ul>
+          <li>HTML</li>
+          <li>XML</li>
+          <li>CSS</li>
+        </ul>
+        <a href="#" class="btn-text">Get my resume</a>*
+      <br> 
+      &nbsp;
+      </article>
+    </div>
+
+
+    <div class="achievement">
+      <div class="img">
+        <img src="./images/Snapshoot3.png" alt="" srcset="">
+      </div>
+      <article>
+        <h3>Tonic</h3>
+        <p class="infos">
+          <span>canopy</span> • Back End Dev • 2015
+        </p>
+        <p class="prg">A daily selection of privately personalized reads; no accounts or sign-ups required..</p>
+        <ul>
+          <li>HTML</li>
+          <li>XML</li>
+          <li>CSS</li>
+        </ul>
+        <a href="#" class="btn-text">Get my resume</a>*
```

Comment:
- [ ] You did great using the `a tag` for your button and also implementing the button interactions. 👍 But I think there are some improvements needed here.
- - The text content of your button is different from the figma design, Please consider changing this. The correct text is `See Project`. 😄 
- Your UI: 
![image](https://user-images.githubusercontent.com/107632780/229258133-363bee86-86de-4d04-86ea-717713808cf2.png)

- Figma UI:
![image](https://user-images.githubusercontent.com/107632780/229258145-2402423e-ffaa-4ead-b3e1-ef08f13e31db.png)

- - Also you have an `astricts` after your `See Project buttons` please remove this as it's not part of the design. Thanks